net-im/vacuum/vacuum-9999.ebuild

```diff
@@ -14,9 +13,9 @@ HOMEPAGE="http://www.vacuum-im.org/"
 LICENSE="GPL-3"
 SLOT="0/37" # subslot = libvacuumutils soname version
 KEYWORDS=""
-PLUGINS=( adiummessagestyle annotations autostatus avatars birthdayreminder bitsofbinary bookmarks captchaforms chatstates clientinfo commands compress console dataforms datastreamsmanager emoticons filemessagearchive filestreamsmanager filetransfer gateways inbandstreams iqauth jabbersearch messagearchiver messagecarbons multiuserchat pepmanager privacylists privatestorage recentcontacts registration remotecontrol rosteritemexchange rostersearch servermessagearchive servicediscovery sessionnegotiation shortcutmanager socksstreams urlprocessor vcard xmppuriqueries )
+PLUGINS=( annotations autostatus avatars birthdayreminder bitsofbinary bookmarks captchaforms chatstates clientinfo commands compress console dataforms datastreamsmanager emoticons filemessagearchive filestreamsmanager filetransfer gateways inbandstreams iqauth jabbersearch messagearchiver messagecarbons multiuserchat pepmanager privacylists privatestorage recentcontacts registration remotecontrol rosteritemexchange rostersearch servermessagearchive servicediscovery sessionnegotiation shortcutmanager socksstreams urlprocessor vcard xmppuriqueries )
 SPELLCHECKER_BACKENDS="aspell +enchant hunspell"
-IUSE="${PLUGINS[@]/#/+} ${SPELLCHECKER_BACKENDS} +spell"
+IUSE="adiummessagestyle ${PLUGINS[@]/#/+} ${SPELLCHECKER_BACKENDS} +spell"
```

Comment:
Is it just me or does this change actually nothing? Unless the build switch is somehow broken if treated like all the others?

Reply:
You can reproduce this if build with `USE="-adiummessagestyle"`:
```
CMake Warning:
  Manually-specified variables were not used by the project:

    PLUGIN_adiummessagestyle
```

Reply:
I see. However, it seems by deleting `-DNO_WEBKIT=$(usex !adiummessagestyle)` you could get away with it more easily.

Reply:
Really to delete this line? But what about the dependence on webkit? I think it's more dirty.

Reply:
Well it only searches for `find_package (Qt5WebKit)` (no 'required' keyword) so it is optional, there won't be a dependency. You could still disable searching for it via `-DCMAKE_FIND_PACKAGE_Qt5WebKit=$(usex adiummessagestyle).` `NO_WEBKIT` is actually nowhere used throughout the code or in other cmake files, because adiummessagestyle directory is simply conditional on `Qt5WebKit_FOUND`. Inside the plugin directory we suddenly find another `find_package(Qt5WebKitWidgets`, so it does not even meet the goal of keeping all dependency searching in root CMakeLists.txt file. All the checks could be dropped and the plugin be disabled like the others if `find_package (Qt5WebKit)` were simply done inside the plugin dir.

So upstream cmake could use some cleaning up. If you want to be nice to upstream you could help them fix that, otherwise I am fine with merging this as-is.

Reply:
I tried to make `-DCMAKE_DISABLE_FIND_PACKAGE_Qt5WebKit=$(usex !adiummessagestyle)` but `PLUGIN_adiummessagestyle` still shown as manually-specified. :(

Reply:
yeah, it's the same outcome, I forgot to mention that - it was just to show that this `NO_WEBKIT` flag upstream is redundant upstream.

Reply:
Vacuum-Im developers now agree to drop NO_WEBKIT. May be I'll revert this changes and drop it here. (And commit snapshot ebuild after this)